src/ui/components/LeagueTopBar.tsx

```diff
@@ -62,6 +65,37 @@ const LeagueTopBar = memo(() => {
 		};
 	}, [updateNumberOfScoreBoxes]);
 
+	useEffect(() => {
+		const leagueTopBar = leagueTopBarRef.current;
+		if (
+			!leagueTopBar ||
+			!show ||
+			leagueTopBar.scrollWidth <= leagueTopBar.clientWidth
```

Comment:
What's this check for?

Reply:
- `!leagueTopBar` ensures that `leagueTopBarRef.current` actually exists. On the initial render it doesn't exist for some reason (At least in my testing)
- `!show` is used to make sure the useEffect only runs when the leagueTopBar exists. (No point in adding an event listener if the topBar isn't visible)
- `leagueTopBar.scrollWidth <= leagueTopBar.clientWidth` is used to ensure that scrolling will remain normal when there is no overflow present. I put this line of code in because I couldn't scroll down when there was only one or two games and my mouse was hovering over the leagueTopBar

Reply:
Okay, the scrollWidth/clientWidth check should now go inside the `handleWheel`, since useEffect will no longer be run every render.